index.html

```diff
@@ -26,22 +27,23 @@ <h1>Welcome to Alison's CSD120 Website!</h1>
         <p> My other passion is DND or Dungeons and Dragons. The creative element combined with the teamwork skills it develops and just straight up comedy that can come from one singular sessions pulls me back over and over again. I ran my highschool's club for three years and I plan to run another here.
         </p>
             <h2>My Three Favourite Classes In DnD</h2>
-            <ul>
-            <li>Cleric, tank and heals means best of both worlds</li>
-            <li>Barbarian, means little rp but lots of damage</li>
-            <li>Rogue, lots of rp oppritunities plus edgy backstory bonus points</li>
-            </ul>
+            <ol>
+                <li>Cleric, tank and heals means best of both worlds</li>
+                <li>Barbarian, means little rp but lots of damage</li>
+                <li>Rogue, lots of rp oppritunities plus edgy backstory bonus points</li>
+            </ol>
         <h2>Haiku</h2>
-        <p><br>On this road</br>
+        <p>On this road</br>
             where nobody else travels
             <br>autumn nightfall</br>
-            by <b>Basho</b></p>
```

Comment:
Note the distinction between the `<em>` element and the `<b>` (or `<strong>`) element.  The author line was to be emphasized with just the author strong